test/Libraries/DynamoPythonTests/PythonTestsWithLogging.cs

```diff
@@ -70,61 +70,57 @@ public void ResetCypythonLogsToConsoleAfterRun()
+Python Script: considering clr._extras
+Python Script: considering Autodesk
+Python Script: considering Autodesk.DesignScript
+Python Script: considering Autodesk.DesignScript.Geometry
+Python Script: considering importlib._bootstrap
+Python Script: considering importlib._bootstrap_external
+Python Script: considering types
+Python Script: considering importlib
+Python Script: considering importlib.machinery
 Python Script: considering _collections_abc
-Python Script: considering _operator
-Python Script: considering operator
-Python Script: considering keyword
 Python Script: considering _heapq
 Python Script: considering heapq
 Python Script: considering itertools
+Python Script: considering keyword
+Python Script: considering _operator
+Python Script: considering operator
 Python Script: considering reprlib
 Python Script: considering _collections
 Python Script: considering collections
-Python Script: considering _stat
-Python Script: considering stat
-Python Script: considering genericpath
-Python Script: considering ntpath
-Python Script: considering os.path
-Python Script: considering os
-Python Script: considering types
+Python Script: considering collections.abc
+Python Script: considering _functools
+Python Script: considering functools
+Python Script: considering contextlib
 Python Script: considering enum
 Python Script: considering _sre
 Python Script: considering sre_constants
 Python Script: considering sre_parse
 Python Script: considering sre_compile
-Python Script: considering _functools
-Python Script: considering functools
 Python Script: considering _locale
 Python Script: considering copyreg
 Python Script: considering re
-Python Script: considering platform
-Python Script: considering _socket
-Python Script: considering collections.abc
-Python Script: considering math
-Python Script: considering select
-Python Script: considering selectors
-Python Script: considering errno
-Python Script: considering socket
-Python Script: considering warnings
-Python Script: considering CLR
-Python Script: considering clr
-Python Script: considering atexit
-Python Script: considering clr._extras
-Python Script: considering Autodesk
-Python Script: considering Autodesk.DesignScript
-Python Script: considering Autodesk.DesignScript.Geometry
-Python Script: considering importlib._bootstrap
-Python Script: considering importlib._bootstrap_external
-Python Script: considering importlib
-Python Script: considering importlib.machinery
+Python Script: considering typing.io
+Python Script: considering typing.re
+Python Script: considering typing
 Python Script: considering importlib.abc
-Python Script: considering contextlib
-Python Script: considering importlib.util";
+Python Script: considering importlib.util
+Python Script: considering _stat
+Python Script: considering stat
+Python Script: considering genericpath
+Python Script: considering ntpath
+Python Script: considering os.path
+Python Script: considering os";
```

Comment:
For some reason, this list of CPython modules is now different because of which this test began failing. I've updated it with a new list now but not sure why this would have changed if it's expected to.